lib/b2c/users.ts

```diff
@@ -1,43 +1,657 @@
+  biometric_registrations: BiometricRegistration[];
+  // The name of the User. Each field in the `name` object is optional.
+  name?: Name;
+  /**
+   * The timestamp of the User's creation. Values conform to the RFC 3339 standard and are expressed in UTC,
+   * e.g. `2021-12-29T12:33:09Z`.
+   */
+  created_at?: string;
+  // The password object is returned for users with a password.
+  password?: Password;
+  /**
+   * The `trusted_metadata` field contains an arbitrary JSON object of application-specific data. See the
+   * [Metadata](https://stytch.com/docs/api/metadata) reference for complete field behavior details.
+   */
+  trusted_metadata?: Record<string, any>; // eslint-disable-line @typescript-eslint/no-explicit-any
+  /**
+   * The `untrusted_metadata` field contains an arbitrary JSON object of application-specific data. Untrusted
+   * metadata can be edited by end users directly via the SDK, and **cannot be used to store critical
+   * information.** See the [Metadata](https://stytch.com/docs/api/metadata) reference for complete field
+   * behavior details.
+   */
+  untrusted_metadata?: Record<string, any>; // eslint-disable-line @typescript-eslint/no-explicit-any
+}
+
+export interface UsersResultsMetadata {
+  // The total number of results returned by your search query.
+  total: number;
+  /**
+   * The `next_cursor` string is returned when your search result contains more than one page of results.
+   * This value is passed into your next search call in the `cursor` field.
+   */
+  next_cursor?: string;
+}
+
+export interface WebAuthnRegistration {
+  // The unique ID for the WebAuthn registration.
+  webauthn_registration_id: string;
+  // The `domain` on which a WebAuthn registration was started. This will be the domain of your app.
+  domain: string;
+  // The user agent of the User.
+  user_agent: string;
+  /**
+   * The verified boolean denotes whether or not this send method, e.g. phone number, email address, etc.,
+   * has been successfully authenticated by the User.
+   */
+  verified: boolean;
+  /**
+   * The `authenticator_type` string displays the requested authenticator type of the WebAuthn device. The
+   * two valid types are "platform" and "cross-platform". If no value is present, the WebAuthn device was
+   * created without an authenticator type preference.
+   */
+  authenticator_type: string;
+}
+
+// Request type for `users.create`.
+export interface UsersCreateRequest {
+  // The email address of the end user.
   email?: string;
-  phone_number?: string;
+  // The name of the user. Each field in the name object is optional.
   name?: Name;
-  create_user_as_pending?: boolean;
+  // Provided attributes help with fraud detection.
   attributes?: Attributes;
-  trusted_metadata?: UserMetadata;
-  untrusted_metadata?: UserMetadata;
+  /**
+   * The phone number to use for one-time passcodes. The phone number should be in E.164 format. The phone
```

Comment:
This comment seems to be more a consequence of us reusing the same `phone_number` param in docs rather than being applicable to CreateUser. I think its fine for now but probably a bit out of context. For example, I think our `CreateUser` sandbox only checks for a sandbox email https://github.com/stytchauth/api/blob/main/pkg/middleware/sandbox/response/response.go#L108